atc/integration/rbac_test.go

```diff
@@ -90,6 +90,37 @@ jobs:
 				})
 			})
 
+			Context("when the role is pilot", func() {
+				BeforeEach(func() {
+					team.Auth = atc.TeamAuth{
+						"pilot": map[string][]string{
+							"users":  []string{"local:o-user"},
```

Comment:
```suggestion
							"users":  []string{"local:p-user"},
```

Following the integration test role user conventions

Reply:
oof... nice spot. not a fan of that single letter representative :D

Reply:
:D